tests/network-ovn

```diff
@@ -1,25 +1,42 @@
 #!/bin/bash
 set -eux
 
-# Install dependencies
-install_deps ovn-host ovn-central bind9-dnsutils jq
-
 # Install LXD
 install_lxd
 
-# Configure OVN.
-ovs-vsctl set open_vswitch . \
-  external_ids:ovn-remote=unix:/var/run/ovn/ovnsb_db.sock \
-  external_ids:ovn-encap-type=geneve \
-  external_ids:ovn-encap-ip=127.0.0.1
-
-# Empty controller log so ACL log checks are consistent over repeat runs.
-echo "" > /var/log/ovn/ovn-controller.log
-
 # Configure LXD.
 lxc project switch default
 lxc storage create default zfs
 
+# If the first parameter is "microovn", we use the MicroOVN snap as a backend for the tests.
+if [ "$#" -gt 0 ] && [ "$1" = "microovn" ]; then
+    if [ -n "$2" ]; then
+        microovn_channel="$2"
+        shift 1
+    else
+        microovn_channel="latest/stable"
+    fi
```

Comment:
Is the extra `microovn` argument required? Why not just invoke the test as `tests/network-ovn latest/stable`?

Reply:
It is not stricly required but it is just to distinguish the LXD channel from the MicroOVN channel. I found that
`./bin/local-run tests/network-ovn 5.21/edge microovn 22.03/stable` (this uses `5.21/edge` for LXD and `22.03/stable` for MicroOVN)

is more understandable than:
`./bin/local-run tests/network-ovn 5.21/edge 22.03/stable`

Reply:
Right, that's indeed quite confusing, makes sense to have `microovn` for clarity. Thanks

Reply:
@gabrielmougard thanks for tackling this BTW! Once that lands, I'll probably rework it a bit to shove it under a helper function that relies on a `MICROOVN_SNAP_CHANNEL` env variable, similar to the `install_lxd()` helper.